src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -109,52 +123,60 @@ const FormDefinitionDetailPage = () => {
                   />
                 </CommonLabeledTextBlock>
                 <CommonLabeledTextBlock title='Status'>
-                  <FormStatusText identifer={formIdentifier} />
+                  <FormStatusText identifier={formIdentifier} />
                 </CommonLabeledTextBlock>
               </Stack>
             </CommonCard>
           </Grid>
           <Grid item xs={12} md={4}>
             <CommonCard title='Actions' titleComponent='h5'>
               <Stack gap={1.5}>
-                <RootPermissionsFilter permissions='canManageForms'>
-                  <ButtonLink
-                    to={generatePath(AdminDashboardRoutes.PREVIEW_FORM_DRAFT, {
-                      identifier: formIdentifier.identifier,
-                      formId: formIdentifier.draftVersion?.id || '',
-                    })}
-                    variant='contained'
-                    fullWidth
-                    disabled={!hasDraft}
-                  >
-                    Preview / Publish Draft
-                  </ButtonLink>
-                  <EditFormButton
-                    formIdentifier={formIdentifier}
-                    text={hasDraft ? 'Edit Draft' : 'New Draft'}
-                    editorType={FormEditorType.FormBuilder}
-                    variant='outlined'
-                  />
-                  <RootPermissionsFilter permissions='canAdministrateConfig'>
-                    <EditFormButton
-                      formIdentifier={formIdentifier}
-                      text={hasDraft ? 'Edit Draft (JSON)' : 'New Draft (JSON)'}
-                      editorType={FormEditorType.JsonEditor}
-                      variant='outlined'
-                    />
-                  </RootPermissionsFilter>
-                </RootPermissionsFilter>
                 <ButtonLink
                   to={generatePath(AdminDashboardRoutes.PREVIEW_FORM, {
                     identifier: formIdentifier.identifier,
                     formId: formIdentifier.displayVersion.id,
                   })}
-                  variant='outlined'
+                  variant={hasDraft ? 'outlined' : 'contained'}
```

Comment:
Suggestion from me: When there is a draft, the primary action is "Preview / Publish Draft". When there's no draft, the primary action is "View Published Form".
Background: [Figma](https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP?node-id=1125-8806#921966539), related [Slack](https://greenriver.slack.com/archives/C033YNSN0Q1/p1724174821969259)